boucle.py

```diff
@@ -4,18 +4,19 @@
 
 import jack
 import sys
-from clip import Clip, Song
+from clip import Clip, Song, getDefaultOutputNames
 from gui import Gui
 from PyQt5.QtWidgets import QApplication
 from queue import Empty
 
-song = Song(8, 8)
+song = Song(8, 8, getDefaultOutputNames(8))
 
 client = jack.Client("Super Boucle")
 midi_in = client.midi_inports.register("input")
 midi_out = client.midi_outports.register("output")
-outL = client.outports.register("output_L")
-outR = client.outports.register("output_R")
+outL = client.outports.register("MAIN_L")
+outR = client.outports.register("MAIN_R")
+
 inL = client.inports.register("input_L")
 inR = client.inports.register("input_R")
 
```

Comment:
@Vampouille having to add empty clips to all songs that do not use somw of your setups outputs feels a bit like a hack to me - and considering that (you mentioned it yourself) usually the output will be static, it would be one that most users are going to be affected by.

Right now I have implemented everything like you said, I'll commit in a few (just some more testing needed) but I think it might be better to somehow have the option to maintain outputs in a song without the need for clips.
What doy you think?